src/ifElse.test.js

```diff
@@ -1,11 +1,49 @@
 'use strict';
 
+const { ifElse } = require('./ifElse');
+
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  it('should run the first callback if the condition returns true', () => {
+    const condition = jest.fn(() => true);
+    const first = jest.fn();
+    const second = jest.fn();
```

Comment:
what is the first and the second?) you should set to variables understandable names
firstCallback, secondCallback
fix it here and everywhere below